monitor/main.go

```diff
@@ -128,11 +128,17 @@ func Start(ctx context.Context, cfg *Config) error {
 	if err != nil {
 		return err
 	}
+	nextBlock := startBlock
+	if startBlock.Cmp(big.NewInt(0)) != 0 {
```

Comment:
I'm not familiar with the problem, would you mind elaborating on what it is and how this PR solves it?

Reply:
Last persisted block is not the block we want to parse, because we've already parsed it...

Blocks ...4, 5, 6 have been parsed. When we restart eksportisto we don't want to parse 6 again.